src/components/Answer/Answer.js

```diff
@@ -1,11 +1,22 @@
 import Question from '../../model/question';
 
 export default class Answer {
-  timeOfAnswer;
-  Question;
-  lifelineUsed = false;
+  constructor(timeOfAnswer, Question, answer, lifelineUsed) {
+    this.timeOfAnswer = timeOfAnswer;
+    this.Question = Question;
+    this.lifelineUsed = lifelineUsed;
+    this.answer = answer;
+  }
+
+  corectnessCheck() {
```

Comment:
It is recommended to start methods that represent an action to start with a verb. In this case you can use checkIfCorrect